integration_tests/dbt_project.yml

```
@@ -61,6 +63,10 @@ vars:
     quickbooks_vendor_credit_identifier: "vendor_credit_data"
     quickbooks_vendor_identifier: "vendor_data"
 
+
+  # For validation testing. To be commented out before release.
+  # account_type_exclusions: ['Discount']
```

In order for this to work I had to change this variable from using brackets `[]` to using parenthesis `()`.
```suggestion
  # account_type_exclusions: ('Discount')
```

Additionally, I imagine this will not be necessary after this release is live. Would it be relevant to include in the comment a short detail that this exclusion for 'Discount' will not be needed in the release following `v0.14.0`, but can still be leveraged in future updates.

---

Huh, weird it worked for me. But this is edited now.